src/lib/common/JsonHelper.cpp

```diff
@@ -149,6 +154,46 @@ void JsonHelper::addRaw(const std::string& key, const std::string& value)
 }
 
 
+/* ****************************************************************************
+*
+* JsonHelper::addDouble -
```

Comment:
Sounds good.
Might have to add a

    addNumber(const std::string& key, double value)

in the future?